handbook.md

```diff
@@ -125,6 +125,170 @@ We follow the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) an
+* ["no-undefined": "off"](http://eslint.org/docs/rules/no-undefined)
+* ["no-underscore-dangle": "off"](http://eslint.org/docs/rules/no-underscore-dangle)
+* ["no-unneeded-ternary": "error"](http://eslint.org/docs/rules/no-unneeded-ternary)
+* ["no-unreachable": "error"](http://eslint.org/docs/rules/no-unreachable)
+* ["no-unused-expressions": "off"](http://eslint.org/docs/rules/no-unused-expressions)
+* ["no-unused-vars": ["error", { "vars": "all", "args": "none" }]](http://eslint.org/docs/rules/no-unused-vars)
+* ["no-use-before-define": ["error", "nofunc"]](http://eslint.org/docs/rules/no-use-before-define)
+* ["no-var": "error"](http://eslint.org/docs/rules/no-var)
+* ["no-void": "off"](http://eslint.org/docs/rules/no-void)
+* ["no-warning-comments": "off"](http://eslint.org/docs/rules/no-warning-comments)
+* ["no-with": "error"](http://eslint.org/docs/rules/no-with)
+* ["one-var": "off"](http://eslint.org/docs/rules/one-var)
+* ["operator-assignment": "off"](http://eslint.org/docs/rules/operator-assignment)
+* ["operator-linebreak": ["error", "after"]](http://eslint.org/docs/rules/operator-linebreak)
+* ["padded-blocks": "off"](http://eslint.org/docs/rules/padded-blocks)
+* ["quote-props": "off"](http://eslint.org/docs/rules/quote-props)
+* ["quotes": ["error", "single", {"avoidEscape": true}]](http://eslint.org/docs/rules/quotes)
+* ["radix": "error"](http://eslint.org/docs/rules/radix)
+* ["semi": ["error", "always"]](http://eslint.org/docs/rules/semi)
+* ["semi-spacing": "off"](http://eslint.org/docs/rules/semi-spacing)
+* ["sort-vars": "off"](http://eslint.org/docs/rules/sort-vars)
+* ["keyword-spacing": ["error", {"before": true, "after": true}]](http://eslint.org/docs/rules/keyword-spacing)
+* ["space-before-blocks": ["error", "always"]](http://eslint.org/docs/rules/space-before-blocks)
+* ["space-before-function-paren": ["error", {"anonymous": "always", "named": "never"}]](http://eslint.org/docs/rules/space-before-function-paren)
+* ["space-in-brackets": "off"](http://eslint.org/docs/rules/space-in-brackets)
+* ["space-in-parens": ["error", "never"]](http://eslint.org/docs/rules/space-in-parens)
+* ["space-infix-ops": "error"](http://eslint.org/docs/rules/space-infix-ops)
+* ["space-unary-ops": ["error", { "words": true, "nonwords": false }]](http://eslint.org/docs/rules/space-unary-ops)
+* ["spaced-comment": ["error", "always"]](http://eslint.org/docs/rules/spaced-comment)
+* ["strict": "off"](http://eslint.org/docs/rules/strict)
+* ["use-isnan": "error"](http://eslint.org/docs/rules/use-isnan)
+* ["valid-jsdoc": "off"](http://eslint.org/docs/rules/valid-jsdoc)
+* ["valid-typeof": "error"](http://eslint.org/docs/rules/valid-typeof)
+* ["vars-on-top": "error"](http://eslint.org/docs/rules/vars-on-top)
+* ["wrap-iife": ["error", "any"]](http://eslint.org/docs/rules/wrap-iife)
+* ["wrap-regex": "off"](http://eslint.org/docs/rules/wrap-regex)
+* ["yoda": ["error", "never"]](http://eslint.org/docs/rules/yoda)
```

Comment:
This list will quickly become obsolete as we will be modifying jslint configuration file. Can you provide a link to the configuration stored in git preferably instead?